pkg/odo/cli/pipelines/service/add.go

```diff
@@ -75,7 +75,8 @@ func newCmdAdd(name, fullName string) *cobra.Command {
 	cmd.Flags().StringVar(&o.ImageRepo, "image-repo", "", "used to push built images")
 	cmd.Flags().StringVar(&o.InternalRegistryHostname, "internal-registry-hostname", "image-registry.openshift-image-registry.svc:5000", "internal image registry hostname")
 	cmd.Flags().StringVar(&o.PipelinesFilePath, "pipelines-file", "pipelines.yaml", "path to pipelines file")
-	cmd.Flags().StringVarP(&o.SealedSecretsNamespace, "sealed-secrets-ns", "", "kube-system", "namespace in which the Sealed Secrets operator is installed, automatically generated secrets are encrypted with this operator")
+	cmd.Flags().StringVarP(&o.SealedSecretsNamespace, "sealed-secrets-ns", "", "sealed-secrets", "namespace in which the Sealed Secrets operator is installed, automatically generated secrets are encrypted with this operator")
+	cmd.Flags().StringVarP(&o.SealedSecretsController, "sealed-secrets-controller", "", "sealedsecretcontroller-sealed-secrets", "SealedSecrets Controller name (default: sealedsecretcontroller-sealed-secrets)")
```

Comment:
I think that instead of two components, we should use a `types.NamespacedName` here, the two values are now indivisible, this would also reduce the number of parameters being passed around.

Basically this https://refactoring.com/catalog/introduceParameterObject.html

Reply:
Changed to NamespacedName.